features/issues/types/issue.types.ts

```diff
@@ -1,16 +1,31 @@
+import { ProjectLanguage } from "@features/projects";
+
+export enum IssueStatus {
+  Unresolved = "open",
+  Resolved = "resolved",
+}
+
 export enum IssueLevel {
   info = "info",
   warning = "warning",
   error = "error",
 }
 
+interface ProjectName {
+  [ProjectLanguage.react]: "frontend - web";
+  [ProjectLanguage.node]: "backend";
+  [ProjectLanguage.python]: "ml service";
+}
```

Comment:
This is problematic unfortunately. The project names "frontend - web" and so on are dynamic data that come from the server. So defining them will cause problems when there are other projects.